src/main/java/org/junit/contrib/java/lang/system/internal/NoExitSecurityManager.java

```diff
@@ -10,30 +14,43 @@
  * delegated to the original security manager.
  */
 public class NoExitSecurityManager extends SecurityManager {
+
 	private final SecurityManager originalSecurityManager;
+	private final SynchLatch latch;
+
 	private Integer statusOfFirstExitCall = null;
 
 	public NoExitSecurityManager(SecurityManager originalSecurityManager) {
+		this(originalSecurityManager, new EmptySynchLatch());
```

Comment:
Please delete this constructor because we don't need it anymore. It is only used by the test and you can use `new NoSecurityManager(..., 0)` instead.